lib/tasks/questionnaires.rake

```diff
@@ -0,0 +1,16 @@
+namespace :questionnaires do
+  desc "Initialize questionnaire responses"
+  task initialize: :environment do
+    date = CountryConfig.current_country?("Ethiopia") ? Date.current : 1.month.ago
```

Comment:
Shouldn't this be one month later? 

In the current month, I want to initialize the questionnaire for the next month?

Reply:
For future, wondering if the initialization config should be a part of the questionnaire itself.

Reply:
> In the current month, I want to initialize the questionnaire for the next month?

@govindkrjoshi For all countries, we initialize questionnaire for previous month in current month. That leads to a delay of 20-26 days in Ethiopia. Initializing for current month negates that and users will start seeing the forms 5-10 days in advance depending on the month.

Reply:
> wondering if the initialization config should be a part of the questionnaire itself.

Good suggestion. This logic can/must be moved from code to data. We need 2 pieces of information: `when to initialize` and `for which month-date to initialze`. This can be represented as such
```ruby
# When: Every 1 month at 6 AM
# month-date = -1/0/1.month.from_now
questionnaire.init_config = {when: "0 6 1 * *", relative_month_date: 0}```

However, I'm not sure how to generate a config during runtime that `whenever` gem understands and puts in crontab. Will need to spike that unknown part.